### ISSUE TITLE
Make economic fee more usable

### ISSUE THREAD

**Issue by Per9** [bitcoin-wallet/bitcoin-wallet#711]
The current options are normal which currently costs a whooping $5 for one more or less immediate transaction, or economic which is almost nothing but will probably never confirm unless you give in and pay the $20 it costs to raise the fee. You could for instance make economic mode a configurable percentage of the the normal mode, a configurable fixed number, or let the user enter a number for each transaction.

**Comment by schildbach** [comment]
First of all, all transactions with economic fee **do** confirm, without a fee raise. It just takes a couple of weeks when the network is overloaded.

The relations/distances of the several fee levels change frequently, so I fail to see the meaning behind a fixed percentage. Can you elaborate on the rationale?

What do you mean by "number"? Sat/vbyte? That has been suggested many times, but I think it's way too advanced – people have no clue what this is. Bitcoin Wallet is aimed at novice users, so we try to offer fee levels based on typical usecases.

**Comment by Per9** [comment]
Do you really think novice users will understand that "economic" can mean weeks and that they most likely won't be able to do any more transactions until then? Or that accepting to wait weeks for a transaction is a typical use case?

How about using https://bitcoinfees.earn.com/api to let the user select an acceptable wait period?

**Comment by schildbach** [comment]
Yes, I think everybody can understand the sentence "Use economic fee only if time is not important".

The typical usecase is sending to yourself (e.g. into a cold wallet) or donations.

**Comment by Per9** [comment]
If "novice" users understand the implications of Bitcoin transactions that well then they will have no problems setting a more sane fee that won't leave it and the wallet stuck until it's removed from the memory pool, and even then it can't be safely ignored. No, "time is not important" does not equal "maybe it'll work, maybe it won't" to most people.

**Comment by OmlineEditor** [comment]
> What do you mean by "number"? Sat/vbyte? That has been suggested many times, but I think it's way too advanced – people have no clue what this is. Bitcoin Wallet is aimed at novice users, so we try to offer fee levels based on typical usecases.

make advanced settings, it is very inconvenient to use your program without manually specifying the commission.
